extension-config.json

```diff
@@ -68,6 +68,17 @@
         "key": "value"
       }
     },
+    "webCheckoutShopify": {
+      "type": "bigApi",
+      "destination" : "frontend",
+      "default": null,
+      "params": {
+        "method":  "GET",
+        "service": "config",
+        "path": "/v1/shop/%(shopId)s/checkout_web_checkout_shopify",
```

Comment:
Please add `?parsed=true` to the url, so that the value of the shop setting is parsed properly. And you can remove the `if (typeof appConfig.webCheckoutShopify === 'string')` block from `pwa-webcheckout-shopify/selectors/index.js`

Reply:
Changed